lib/trusted-apps.js

```diff
@@ -33,13 +33,16 @@ function getReqHeaderValue(req, headerKey) {
 }
 
 function getRequestUrl(req) {
-  if (getReqHeaderValue(req.header, 'X-Requested-URI')) {
-    return getReqHeaderValue(req.header, 'X-Requested-URI');
+  let result = "";
+  if (getReqHeaderValue(req, 'X-Requested-URI')) {
+    result = getReqHeaderValue(req, 'X-Requested-URI');
   } else {
-    return req.protocol + '://' + getReqHeaderValue(req.header, 'Host') + req.url;
+    result = req.protocol + '://' + getReqHeaderValue(req, 'Host') + req.url;
   }
+  return result.replace(/([a-z]+):\/([^\/])/, '$1://$2'); // Regex because of sitebuilder requests for lighthouse reports
```

Comment:
Doesn't seem right to have very pagechecker specific code in this library - at least not by default. Could be a config option?